jsdrupal/drupal-admin-ui
# Improve focus functionality of EntityReferenceAutocomplete

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/266

## Screenshot / UI changes

N/A

## Testing instructions

Visit /node/add/recipe – click any of the autocomplete entity reference fields (Recipe category, or Tags) and begin typing. Focus should remain on the text field consistently while typing.
